hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFsHAURLs.java

```diff
@@ -299,14 +296,14 @@ public void testWithQualifiedDefaultFS() throws Exception {
           new GenericTestUtils.SystemErrCapturer()) {
         res = ToolRunner.run(shell, new String[] {"-ls", unqualifiedPath2});
         // Check stderr
-        Assert.assertThat("Command did not print the error message " +
+        MatcherAssert.assertThat("Command did not print the error message " +
                 "correctly for test case: "
                 + "ozone fs -ls o3fs://bucket.volume.id1:port/",
             capture.getOutput(), StringContains.containsString(
                 "does not use port information"));
       }
       // Check return value, should be -1 (failure)
-      Assert.assertEquals(res, -1);
+      assertEquals(-1, res);
```

Comment:
Nice! (swapping arguments to reflect `expected, actual` order)

Reply:
Thanks.